beacon-chain/db/slasherkv/slasher.go

```diff
@@ -166,7 +166,7 @@ func (s *Store) CheckAttesterDoubleVotes(
 					validatorEpochKey := append(encEpoch, encIdx...)
 					attRecordsKey := signingRootsBkt.Get(validatorEpochKey)
 
-					// An attestation record key is comprised of a signing root (32 bytes).
+					// An attestation record key is compromised of a signing root (32 bytes).
```

Comment:
why `compromised`?

Reply:
Sorry, my bad, it applies that auto suggestion from my spelling corrector. Correcting to consists of.